tasks/91_make-a-person/make-a-person.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Class to create a person
+ */
+class Person {
+  /**
+   * constructor function used to create new instances of Person
+   * @param {String} fullName, person's first and last name
+   */
+  constructor(fullName) {
+    this.setFirstName(fullName.split(' ')[0]);
```

Comment:
What if signature of `setFirstName` will be changed? It is better to explicitly assign values in constructor

Reply:
Agree